config/locales/en.yml

```diff
@@ -815,6 +815,10 @@ en:
       you_have_not_yet_accepted_or_rejected_offer: "You have not yet accepted or rejected the offer %{title} you received %{date}."
       you_have_not_yet_accepted_or_rejected_request: "You have not yet accepted or rejected the request %{title} you received %{date}."
       show_thread: "Show conversation"
+    branding:
```

Comment:
Do we want to have there in the `en.yml`, which means that translators can translate this?